packages/graph/hash_graph/tests/integration/postgres/data_type.rs

```diff
@@ -0,0 +1,54 @@
+use crate::postgres::DatabaseTestWrapper;
+
+#[test]
+fn insert() {
+    let boolean_dt = serde_json::from_str(crate::test_data::data_type::BOOLEAN_V1)
+        .expect("could not parse data type");
+
+    let mut database = DatabaseTestWrapper::new();
+    database
+        .create_data_type(boolean_dt)
+        .expect("could not create data type");
+}
+
+#[test]
+fn query() {
+    let empty_list_dt = serde_json::from_str(crate::test_data::data_type::EMPTY_LIST_V1)
+        .expect("could not parse data type");
+
+    let mut database = DatabaseTestWrapper::new();
+
+    let created_data_type = database
+        .create_data_type(empty_list_dt)
+        .expect("could not create data type");
+
+    let data_type = database
+        .get_data_type(created_data_type.version_id())
+        .expect("could not query data type");
```

Comment:
This will drop the wrapper as expected. If you `panic` while dropping it will try to drop. If you `panic` while you `panic`, Rust bails and aborts. 😄